doc/api/esm.md

```diff
@@ -227,204 +167,11 @@ package. It is not a strong encapsulation since a direct require of any
 absolute subpath of the package such as
 `require('/path/to/node_modules/pkg/subpath.js')` will still load `subpath.js`.
 
-#### Subpath Exports
```

Comment:
Should there be a link from here or some other indication that users should seek content related to ESM exports in the Package docs?